docs/modules/ROOT/pages/connect-outside-kubernetes.adoc

```diff
@@ -57,6 +57,9 @@ To allow connections from clients, you need to configure the `exposeExternally`
 - `LoadBalancer` (default): Hazelcast client configuration `<load-balancer-external-ip>:5701`.
 +
 WARNING: Your Kubernetes cluster should be able to assign public IPs to LoadBalancer services.
+
+WARNING: Expose externally feature is not supported in Istio setup.
```

Comment:
Two warnings one after another is not a good look, I have added a comment to combine both warnings, so lines 59 an 60 will need to be removed.

Reply:
ohh, I forgot about that

Reply:
Actually, I think the second warning should be an IMPORTANT admonition, and belongs at the top of the file; so,

Line 42, put a new paragraph after the first sentence, then the IMPORTANT admonition, then another new para for the introduction to the table. Like this:

```
== Configuring `exposeExternally`

To allow connections from clients, you need to configure the `exposeExternally` field, of the Hazelcast custom resource. 

IMPORTANT: If using Itsio, you cannot use `exposeExternally`.

This field has the following properties:
```
Is there something else that they can use for the same effect?

Reply:
no unfortunately